‚è≥ Waiting for DBus to be ready...
‚úÖ DBus is ready!
üîê Starting SWTPM...
‚úÖ SWTPM is ready!
‚öôÔ∏è  Setting up TPM-backed Yggdrasil config in RAM...
üìÅ Read metadata: Primary=0x81014cc9, Key=0x810143d5
üîÅ Using previously stored handles
‚úÖ Primary handle exists: 0x81014cc9
üóòÔ∏è Generating Yggdrasil config in /dev/shm...
‚úÖ Key handle exists: 0x810143d5
Storing metadata: Primary=0x81014cc9, Key=0x810143d5
Metadata stored at /tpmdata/handles.json:
{
  "primary_handle": "0x81014cc9",
  "key_handle": "0x810143d5"
}
üîì Unsealing Yggdrasil key from TPM...
Injecting private key into Yggdrasil config...
üöÄ Launching Yggdrasil with in-memory config...
2025/04/22 23:51:30 Build name: yggdrasil
2025/04/22 23:51:30 Build version: 0.5.12
2025/04/22 23:51:30 Your public key is ec461538d8f2c7d165ccf228473c63e1e5ad92c888ff0b4f85e005f34475d8f9
2025/04/22 23:51:30 Your IPv6 address is 200:2773:d58e:4e1a:705d:3466:1baf:7187
2025/04/22 23:51:30 Your IPv6 subnet is 300:2773:d58e:4e1a::/64
2025/04/22 23:51:30 UNIX admin socket listening on /var/run/yggdrasil.sock
2025/04/22 23:51:30 TLS listener started on [fe80::834f:d15:9c1c:6152%wlo1]:33271
2025/04/22 23:51:30 Interface name: tun2
2025/04/22 23:51:30 Interface IPv6: 200:2773:d58e:4e1a:705d:3466:1baf:7187/7
2025/04/22 23:51:30 Interface MTU: 65535
2025/04/22 23:51:30 Connected inbound: 201:bbbb:b793:7a64:2657:ae3b:ae38:c33@[fe80::834f:d15:9c1c:6152%wlo1]:38705, source [fe80::834f:d15:9c1c:6152%wlo1]:33271
2025/04/22 23:51:30 Connected inbound: 201:2bd3:1946:41ea:d21a:a129:2b2f:c3a6@[fe80::334e:e55:c285:b3ff%wlo1]:42237, source [fe80::834f:d15:9c1c:6152%wlo1]:33271
2025/04/22 23:51:30 Connected inbound: 201:2bd3:1946:41ea:d21a:a129:2b2f:c3a6@[fe80::5931:ab55:cc59:4b%wlo1]:33367, source [fe80::834f:d15:9c1c:6152%wlo1]:33271
‚úÖ Yggdrasil started with IPv6: address:
subnet:
üîç Discovering peers...
                     URI                     	State	Dir	              IP Address              	Uptime	  RTT  	 RX  	 TX  	 Down  	  Up   	Pr	Cost	Last Error 
tls://[fe80::834f:d15:9c1c:6152%25wlo1]:38705	Up   	In 	201:bbbb:b793:7a64:2657:ae3b:ae38:c33 	1s    	4.03ms 	2.5KB	0.3KB	2.5KB/s	0.2KB/s	0 	8   	-         	
tls://[fe80::5931:ab55:cc59:4b%25wlo1]:33367 	Up   	In 	201:2bd3:1946:41ea:d21a:a129:2b2f:c3a6	0s    	8.29ms 	2.5KB	2.5KB	2.5KB/s	2.5KB/s	0 	16  	-         	
tls://[fe80::334e:e55:c285:b3ff%25wlo1]:42237	Up   	In 	201:2bd3:1946:41ea:d21a:a129:2b2f:c3a6	1s    	10.48ms	2.5KB	0.2KB	2.5KB/s	0.2KB/s	0 	20  	-         	
üöÄ Starting RabbitMQ...
‚è≥ Waiting for RabbitMQ to start...

BOOT FAILED
2025-04-22 23:51:32.244472+00:00 [error] <0.156.0> 
2025-04-22 23:51:32.244472+00:00 [error] <0.156.0> BOOT FAILED
2025-04-22 23:51:32.244472+00:00 [error] <0.156.0> ===========
2025-04-22 23:51:32.244472+00:00 [error] <0.156.0> ERROR: could not bind to distribution port 25672, it is in use by another node: rabbit@localhost
2025-04-22 23:51:32.244472+00:00 [error] <0.156.0> 
===========
ERROR: could not bind to distribution port 25672, it is in use by another node: rabbit@localhost

2025-04-22 23:51:33.249644+00:00 [error] <0.156.0>     supervisor: {local,rabbit_prelaunch_sup}
2025-04-22 23:51:33.249644+00:00 [error] <0.156.0>     errorContext: start_error
2025-04-22 23:51:33.249644+00:00 [error] <0.156.0>     reason: {dist_port_already_used,25672,"rabbit","localhost"}
2025-04-22 23:51:33.249644+00:00 [error] <0.156.0>     offender: [{pid,undefined},
2025-04-22 23:51:33.249644+00:00 [error] <0.156.0>                {id,prelaunch},
2025-04-22 23:51:33.249644+00:00 [error] <0.156.0>                {mfargs,{rabbit_prelaunch,run_prelaunch_first_phase,[]}},
2025-04-22 23:51:33.249644+00:00 [error] <0.156.0>                {restart_type,transient},
2025-04-22 23:51:33.249644+00:00 [error] <0.156.0>                {significant,false},
2025-04-22 23:51:33.249644+00:00 [error] <0.156.0>                {shutdown,5000},
2025-04-22 23:51:33.249644+00:00 [error] <0.156.0>                {child_type,worker}]
2025-04-22 23:51:33.249644+00:00 [error] <0.156.0> 
2025-04-22 23:51:33.249851+00:00 [error] <0.154.0>   crasher:
2025-04-22 23:51:33.249851+00:00 [error] <0.154.0>     initial call: application_master:init/4
2025-04-22 23:51:33.249851+00:00 [error] <0.154.0>     pid: <0.154.0>
2025-04-22 23:51:33.249851+00:00 [error] <0.154.0>     registered_name: []
2025-04-22 23:51:33.249851+00:00 [error] <0.154.0>     exception exit: {{shutdown,
2025-04-22 23:51:33.249851+00:00 [error] <0.154.0>                          {failed_to_start_child,prelaunch,
2025-04-22 23:51:33.249851+00:00 [error] <0.154.0>                              {dist_port_already_used,25672,"rabbit",
2025-04-22 23:51:33.249851+00:00 [error] <0.154.0>                                  "localhost"}}},
2025-04-22 23:51:33.249851+00:00 [error] <0.154.0>                      {rabbit_prelaunch_app,start,[normal,[]]}}
2025-04-22 23:51:33.249851+00:00 [error] <0.154.0>       in function  application_master:init/4 (application_master.erl, line 142)
2025-04-22 23:51:33.249851+00:00 [error] <0.154.0>     ancestors: [<0.153.0>]
2025-04-22 23:51:33.249851+00:00 [error] <0.154.0>     message_queue_len: 1
2025-04-22 23:51:33.249851+00:00 [error] <0.154.0>     messages: [{'EXIT',<0.155.0>,normal}]
2025-04-22 23:51:33.249851+00:00 [error] <0.154.0>     links: [<0.153.0>,<0.44.0>]
2025-04-22 23:51:33.249851+00:00 [error] <0.154.0>     dictionary: []
2025-04-22 23:51:33.249851+00:00 [error] <0.154.0>     trap_exit: true
2025-04-22 23:51:33.249851+00:00 [error] <0.154.0>     status: running
2025-04-22 23:51:33.249851+00:00 [error] <0.154.0>     heap_size: 376
2025-04-22 23:51:33.249851+00:00 [error] <0.154.0>     stack_size: 28
2025-04-22 23:51:33.249851+00:00 [error] <0.154.0>     reductions: 163
2025-04-22 23:51:33.249851+00:00 [error] <0.154.0>   neighbours:
2025-04-22 23:51:33.249851+00:00 [error] <0.154.0> 
2025-04-22 23:51:33.252697+00:00 [notice] <0.44.0> Application rabbitmq_prelaunch exited with reason: {{shutdown,{failed_to_start_child,prelaunch,{dist_port_already_used,25672,"rabbit","localhost"}}},{rabbit_prelaunch_app,start,[normal,[]]}}
Kernel pid terminated (application_controller) ("{application_start_failure,rabbitmq_prelaunch,{{shutdown,{failed_to_start_child,prelaunch,{dist_port_already_used,25672,\"rabbit\",\"localhost\"}}},{rabbit_prelaunch_app,start,[normal,[]]}}}")

Crash dump is being written to: erl_crash.dump...done
2025/04/22 23:51:41 Connected outbound: 201:bbbb:b793:7a64:2657:ae3b:ae38:c33@[fe80::834f:d15:9c1c:6152%wlo1]:41741, source [fe80::834f:d15:9c1c:6152%wlo1]:53701
2025/04/22 23:51:55 Connected outbound: 201:2bd3:1946:41ea:d21a:a129:2b2f:c3a6@[fe80::334e:e55:c285:b3ff%wlo1]:43661, source [fe80::834f:d15:9c1c:6152%wlo1]:57127
2025/04/22 23:51:55 Connected outbound: 201:2bd3:1946:41ea:d21a:a129:2b2f:c3a6@[fe80::5931:ab55:cc59:4b%wlo1]:33883, source [fe80::834f:d15:9c1c:6152%wlo1]:41405
